src/commitStrategy/BasicCommitStrategy.ts

```diff
@@ -0,0 +1,116 @@
+import CommitStrategy from './CommitStrategy';
+import IObjectMetadata from '../objects/IObjectMetadata';
+import Commit from '../commits/Commit';
+import CommitOperation from '../commits/CommitOperation';
+
+export const BASIC_COMMIT_STRATEGY = 'basic';
+
+/**
+ * BasicCommitStrategy implements the 'basic' commit strategy
+ */
+export default class BasicCommitStrategy extends CommitStrategy {
+  resolveObject (objectId: string, commits: Commit[]): {
+    /** Resolved object metadata */
+    metadata: IObjectMetadata,
+    /** Resolved object data */
+    data: any;
+  } {
+    // reduce to only those of the right object
+    const basicCommits = this.filterCommits(objectId, BASIC_COMMIT_STRATEGY, commits);
+    // get the latest commit
+    const earliestCommit = BasicCommitStrategy.findEarliestCommit(basicCommits);

```

Comment:
In this case I believe we should use the `create` commit, not just the earliest commit we're aware of. May be able to just do something like `commits.filter(commit.operation === 'create');`